### ISSUE TITLE
Dotterel on Samsung tablet with Ergodox-EZ

### ISSUE THREAD

**Issue by jsampson45** [nimble0/dotterel#10]
Plover on a PC shows a GUI which has buttons "enable" and "disable", and an option to configure it etc. On Dotterel on Samsung Galaxy Tab S4 I have not been able to find any facility to enable or disable the interface. I was able to get the on-screen keyboard to work on one occasion, but I have not been able to get Dotterel to work with a physical keyboard (Ergodox-EZ). I can type QWERTY with the physical keyboard, but not steno. Can anyone advise?

**Comment by nimble0** [comment]
The settings can be changed in the Dotterel app, you should be able to find it by searching your installed apps. From there, go into Machine Settings and enable NKRO under machines. If you go down to key map under NKRO on the same page you can customize the key mapping.

**Comment by jsampson45** [comment]
Is customising the key mapping the same thing as enabling or disabling Dotterel? It is the latter that I want to do - to switch between steno and qwerty. What about chords?

**Comment by nimble0** [comment]
No, the key mapping defines which keyboard key represents which steno key. "Q" and "A" keyboard keys represent the steno key "S-", "W" keyboard key represents the steno key "T-", etc.

You can enable or disable Dotterel by changing the keyboard in Android's settings. While you have an input area focused there should be a notification which says "Change keyboard", just press that and change to Dotterel to enable it or change away to disable it.

> What about chords?

When using a physical keyboard, chording works the same way as Plover.

**Comment by nimble0** [comment]
You need to select Dotterel in the list shown in this screenshot.
![Screenshot_20210528_213415](https://user-images.githubusercontent.com/9959776/120039847-19762d00-bff5-11eb-8a24-de7e5c6dbfbe.png)

Usually when a text box is selected, a notification appears which will open this list. If it doesn't appear then whatever keyboard is active might have a button to open it. I've highlighted the button used on the Google keyboard (this particular button also requires a long press).
